pkg/cronjob/es_rollover.go

```diff
@@ -140,6 +142,18 @@ func esScriptEnvVars(opts v1.Options) []corev1.EnvVar {
 	if val, ok := opts.Map()["es.password"]; ok {
 		envs = append(envs, corev1.EnvVar{Name: "ES_PASSWORD", Value: val})
 	}
+	if val, ok := opts.Map()["es.tls"]; ok {
```

Comment:
At this point, it would probably make sense to create a map between the properties and the env vars and iterate over them. Something like (untested):

```golang
m := opts.Map()
for k, v := range map[string]string{
	"es.index-prefix": "INDEX_PREFIX",
	"es.username": "ES_USERNAME",
	"...": "...",
} {
	if val, ok := m[k]; ok {
		envs = append(envs, corev1.EnvVar{Name: v, Value: val})
	}
}
```

Reply:
Agreed! Will fix